dev/com.ibm.ws.microprofile.metrics/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -0,0 +1,21 @@
+###############################################################################
+# Copyright (c) 2017 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+###############################################################################
+# -------------------------------------------------------------------------------------------------
+#CMVCPATHNAME com.ibm.ws.microprofile.metrics/resources/OSGI-INF/l10n/metatype.properties
+#ISMESSAGEFILE FALSE
+#NLS_ENCODING=UNICODE
+#NLS_MESSAGEFORMAT_NONE
+
+mpMetrics=MicroProfile Metrics
+mpMetrics.desc=Configuration for MicroProfile Metrics.
+
+authentication=Enable authentication 
+authentication.desc=Indicates whether or not users should provide user name and password when accessing metrics endpoint.
```

Comment:
Indicates whether to provide a user name and password when accessing metrics endpoints.